.github/workflows/linux.yaml

```diff
@@ -41,6 +42,48 @@ jobs:
           name: Sources
           path: .tmp
 
+  flatpak:
```

Comment:
Love it, however could we consider moving this to tc from the get-go?
Releng already has support to release to flathub, so we could use that from the getgo 
https://github.com/mozilla-releng/scriptworker-scripts/tree/master/pushflatpakscript

Reply:
I've given it a solid try, but unfortunately we can't get flatpaks building in TC without being able to stand up a privileged container, which will require some work on releng's part. This is due to the need to create sandboxed environments in which the build occurs and is not posssible without extra permissions. (See: https://github.com/flatpak/flatpak/issues/3027)

That being said, it should be pretty easy to push to flathub even without building. IIRC that just amounts to updating a git repository with the YAML manifests.